### ISSUE TITLE
[Bug] position of axis label is not correct when set axis.label.style.direction='vertical'

### ISSUE THREAD

**Issue by da730** [VisActor/VChart#950]
### Version

1.3.1

### Link to Minimal Reproduction

null

### Steps to Reproduce

```typescript
{
    "type": "line",
    "xField": [
        "230914185157013"
    ],
    "yField": [
        "10002"
    ],
    "direction": "vertical",
    "sortDataByAxis": true,
    "seriesField": "20001",
    "padding": 0,
    "labelLayout": "region",
    "data": [
        {
            "id": "data",
            "values": [
                {
                    "10001": "折扣",
                    "10002": "53.40000084042549",
                    "10003": "230914185157019",
                    "20001": "折扣",
                    "230914185157013": "西北",
                    "230914185157019": "53.40000084042549"
                },
                {
                    "10001": "销售额",
                    "10002": "815039.5979347229",
                    "10003": "230914185157022",
                    "20001": "销售额",
                    "230914185157013": "西北",
                    "230914185157022": "815039.5979347229"
                },
                {
                    "10001": "折扣",
                    "10002": "232.15000347048044",
                    "10003": "230914185157019",
                    "20001": "折扣",
                    "230914185157013": "中南",
                    "230914185157019": "232.15000347048044"
                },
                {
                    "10001": "销售额",
                    "10002": "4137415.0951108932",
                    "10003": "230914185157022",
                    "20001": "销售额",
                    "230914185157013": "中南",
                    "230914185157022": "4137415.0951108932"
                },
                {
                    "10001": "折扣",
                    "10002": "234.75000359117985",
                    "10003": "230914185157019",
                    "20001": "折扣",
                    "230914185157013": "东北1",
                    "230914185157019": "234.75000359117985"
                },
                {
                    "10001": "销售额",
                    "10002": "2681567.4745378494",
                    "10003": "230914185157022",
                    "20001": "销售额",
                    "230914185157013": "东北1",
                    "230914185157022": "2681567.4745378494"
                },
                {
                    "10001": "折扣",
                    "10002": "134.2000020891428",
                    "10003": "230914185157019",
                    "20001": "折扣",
                    "230914185157013": "西南",
                    "230914185157019": "134.2000020891428"
                },
                {
                    "10001": "销售额",
                    "10002": "1303124.5089645386",
                    "10003": "230914185157022",
                    "20001": "销售额",
                    "230914185157013": "西南",
                    "230914185157022": "1303124.5089645386"
                },
                {
                    "10001": "折扣",
                    "10002": "331.10000520944595",
                    "10003": "230914185157019",
                    "20001": "折扣",
                    "230914185157013": "华东",
                    "230914185157019": "331.10000520944595"
                },
                {
                    "10001": "销售额",
                    "10002": "4684506.442247391",
                    "10003": "230914185157022",
                    "20001": "销售额",
                    "230914185157013": "华东",
                    "230914185157022": "4684506.442247391"
                },
                {
                    "10001": "折扣",
                    "10002": "74.10000109672546",
                    "10003": "230914185157019",
                    "20001": "折扣",
                    "230914185157013": "华北",
                    "230914185157019": "74.10000109672546"
                },
                {
                    "10001": "销售额",
                    "10002": "2447301.0141382217",
                    "10003": "230914185157022",
                    "20001": "销售额",
                    "230914185157013": "华北",
                    "230914185157022": "2447301.0141382217"
                }
            ],
            "fields": {
                "10001": {
                    "alias": "指标名称 "
                },
                "10002": {
                    "alias": "指标值 "
                },
                "20001": {
                    "alias": "图例项 ",
                    "domain": [
                        "折扣",
                        "销售额"
                    ],
                    "sortIndex": 0,
                    "lockStatisticsByDomain": true
                },
                "230914185157013": {
                    "alias": "地区",
                    "domain": [
                        "东北1",
                        "华北",
                        "华东",
                        "西北",
                        "西南",
                        "中南"
                    ],
                    "sortIndex": 0,
                    "lockStatisticsByDomain": true
                },
                "230914185157019": {
                    "alias": "折扣"
                },
                "230914185157022": {
                    "alias": "销售额"
                }
            }
        }
    ],
    "axes": [
        {
            "type": "band",
            "tick": {
                "visible": false
            },
            "grid": {
                "visible": false,
                "style": {
                    "zIndex": 150,
                    "stroke": "#DADCDD",
                    "lineWidth": 1,
                    "lineDash": [
                        4,
                        2
                    ]
                }
            },
            "orient": "bottom",
            "visible": true,
            "domainLine": {
                "visible": true,
                "style": {
                    "lineWidth": 1,
                    "stroke": "#989999"
                }
            },
            "title": {
                "visible": false,
                "space": 5,
                "text": "地区",
                "style": {
                    "fontSize": 12,
                    "fill": "#363839",
                    "fontWeight": "normal"
                }
            },
            "sampling": false,
            "zIndex": 200,
            "label": {
                "visible": true,
                "space": 4,
                "style": {
                    "fontSize": 12,
                    "fill": "#6F6F6F",
                    "angle": 0,
                    "fontWeight": "normal",
                    "direction": "vertical"
                },
                "autoHide": true,
                "autoHideMethod": "greedy",
                "flush": true
            },
            "hover": true,
            "background": {
                "visible": false,
                "state": {
                    "hover": {
                        "fillOpacity": 0.08,
                        "fill": "#141414"
                    }
                }
            },
            "paddingInner": [
                0.15,
                0.1
            ],
            "paddingOuter": [
                0.075,
                0.075
            ]
        },
        {
            "type": "linear",
            "tick": {
                "visible": false,
                "style": {
                    "stroke": "rgba(255, 255, 255, 0)"
                }
            },
            "niceType": "accurateFirst",
            "zIndex": 200,
            "grid": {
                "visible": true,
                "style": {
                    "zIndex": 150,
                    "stroke": "#DADCDD",
                    "lineWidth": 1,
                    "lineDash": [
                        4,
                        2
                    ]
                }
            },
            "orient": "left",
            "visible": true,
            "domainLine": {
                "visible": true,
                "style": {
                    "lineWidth": 1,
                    "stroke": "rgba(255, 255, 255, 0)"
                }
            },
            "title": {
                "visible": false,
                "text": "折扣",
                "space": 8,
                "style": {
                    "fontSize": 12,
                    "fill": "#363839",
                    "fontWeight": "normal"
                }
            },
            "sampling": false,
            "label": {
                "visible": true,
                "space": 6,
                "flush": true,
                "padding": 0,
                "style": {
                    "fontSize": 12,
                    "maxLineWidth": 174,
                    "fill": "#6F6F6F",
                    "angle": 0,
                    "fontWeight": "normal",
                    "dy": -1,
                    "direction": "vertical"
                },
                "autoHide": true,
                "autoHideMethod": "greedy"
            },
            "hover": true,
            "background": {
                "visible": true,
                "state": {
                    "hover": {
                        "fillOpacity": 0.08,
                        "fill": "#141414"
                    }
                }
            },
            "zero": true,
            "nice": true
        }
    ],
    "color": {
        "field": "20001",
        "type": "ordinal",
        "range": [
            "#2E62F1",
            "#4DC36A"
        ],
        "specified": {},
        "domain": [
            "折扣",
            "销售额"
        ]
    },
    "legends": [
        {
            "type": "discrete",
            "id": "legend",
            "orient": "right",
            "position": "start",
            "layoutType": "normal",
            "visible": true,
            "maxCol": 1,
            "title": {
                "textStyle": {
                    "fontSize": 12,
                    "fill": "#6F6F6F"
                }
            },
            "layoutLevel": 60,
            "item": {
                "focus": true,
                "focusIconStyle": {
                    "size": 14
                },
                "maxWidth": 498,
                "spaceRow": 0,
                "spaceCol": 0,
                "padding": {
                    "top": 1,
                    "bottom": 2,
                    "left": 3,
                    "right": 2
                },
                "background": {
                    "visible": false,
                    "style": {
                        "fillOpacity": 0.001
                    }
                },
                "label": {
                    "style": {
                        "fontSize": 12,
                        "fill": "#6F6F6F"
                    }
                },
                "shape": {
                    "style": {
                        "lineWidth": 0,
                        "symbolType": "square"
                    }
                }
            },
            "pager": {
                "layout": "horizontal",
                "padding": {
                    "left": -18
                },
                "textStyle": {},
                "space": 0,
                "handler": {
                    "preShape": "triangleLeft",
                    "nextShape": "triangleRight",
                    "style": {},
                    "state": {
                        "disable": {}
                    }
                }
            },
            "padding": {
                "top": 0,
                "bottom": 0,
                "left": 16,
                "right": 0
            }
        }
    ],
    "label": {
        "visible": false,
        "overlap": {
            "hideOnHit": true,
            "avoidBaseMark": false,
            "strategy": [
                {
                    "type": "position",
                    "position": [
                        "top",
                        "bottom",
                        "left",
                        "right",
                        "top-right",
                        "top-left",
                        "bottom-right",
                        "bottom-left",
                        "center"
                    ]
                }
            ],
            "clampForce": true
        },
        "style": {
            "fontSize": 12,
            "fontWeight": "normal",
            "zIndex": 100,
            "fill": "#363839",
            "stroke": "rgba(255, 255, 255, 0.8)",
            "lineWidth": 2,
            "strokeOpacity": 1
        },
        "position": "top",
        "smartInvert": false
    },
    "tooltip": {
        "handler": {}
    },
    "point": {
        "style": {
            "shape": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    "circle"
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            },
            "size": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    7.0898154036220635
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            },
            "fill": {
                "field": "20001",
                "type": "ordinal",
                "range": [
                    "#2E62F1",
                    "#4DC36A"
                ],
                "specified": {},
                "domain": [
                    "折扣",
                    "销售额"
                ]
            },
            "stroke": {
                "field": "20001",
                "type": "ordinal",
                "range": [
                    "#2E62F1",
                    "#4DC36A"
                ],
                "specified": {},
                "domain": [
                    "折扣",
                    "销售额"
                ]
            },
            "strokeOpacity": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    1
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            },
            "fillOpacity": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    1
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            }
        },
        "state": {
            "hover": {
                "lineWidth": 2,
                "fillOpacity": 1,
                "strokeOpacity": 1,
                "scaleX": 1.5,
                "scaleY": 1.5
            }
        }
    },
    "seriesMark": "point",
    "line": {
        "style": {
            "curveType": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    "linear"
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            },
            "lineWidth": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    3
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            },
            "lineDash": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    [
                        0,
                        0
                    ]
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            }
        }
    },
    "region": [
        {
            "clip": true
        }
    ],
    "background": "rgba(255, 255, 255, 0)",
    "area": {
        "style": {
            "curveType": {
                "type": "ordinal",
                "field": "20001",
                "range": [
                    "linear"
                ],
                "domain": [
                    "折扣",
                    "销售额"
                ]
            }
        }
    },
    "invalidType": "break",
    "animation": false,
    "hash": "bffc34029f732bfd7639968880dc9a14"
}
```

### Current Behavior

<img width="1075" alt="image" src="https://github.com/VisActor/VChart/assets/20367445/00c4d8a3-1e18-4939-9d4e-e89f076e45f5">

### Expected Behavior

<img width="936" alt="image" src="https://github.com/VisActor/VChart/assets/20367445/693f1c42-08f0-4be0-b70e-85b547b32a43">

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by kkxxkk2019** [comment]
related to vrender: https://github.com/VisActor/VRender/issues/503

**Comment by kkxxkk2019** [comment]
And it will be fixed in VRender@0.15.3
